src/SFA.DAS.AssessorService.Web/Controllers/AccountController.cs

```diff
@@ -51,11 +55,14 @@ public IActionResult SignIn()
         public async Task<IActionResult> PostSignIn()
         { 
             var loginResult = await _loginOrchestrator.Login();
+            var orgName = _contextAccessor.HttpContext.User.FindFirst("http://schemas.portal.com/orgname")?.Value;
```

Comment:
Is it always in the claim? If not then I'd be tempted to use the Orgname property from loginResult

Reply:
Prior to this code being activated the claim is populated with the organisation name which is passed to apply - subsequently assessor didn't use the value it had passed to apply